migration/main.py

```diff
@@ -190,8 +210,14 @@ def run():
     wh_password = os.getenv('WH_PASSWORD')
     logger.info(f'  WH_PASSWORD: *REDACTED*')
 
+    wh_disabled_param = os.getenv('WH_DISABLED', 'false')
+    whDisabled = wh_disabled_param.lower() in ('true', 'yes', '1')
```

Comment:
Should stick with underscored variable style here rather switch to camel case for this one variable.

Reply:
✅ Fixed.